lablib/operators/color.py

```diff
@@ -45,7 +18,7 @@ class ColorOperator(BaseOperator):
     @classmethod
     @abstractmethod
     def from_node_data(cls, data) -> "ColorOperator":
-        """An abstract classmethod for returning a ``ColorOperator`` from node data.
+        """An abstract classmethod for returning a `ColorOperator` from node data.
```

Comment:
i added the double backticks because i had in mind this renders an actual link in the docs but i just built them and they don't. so that's cool

if we want to reference to an actual object we can do so with:
```suggestion
"""An abstract classmethod for returning a :obj:`ColorOperator` from node data.
```